lib/omnibus/software.rb

```diff
@@ -767,6 +746,19 @@ def overridden?
     # @!endgroup
     # --------------------------------------------------
 
+    #
+    # Path to where any source is extracted to.
+    #
+    # Files in a source directory are staged underneath here. Files from
+    # a url are fetched and extracted here. Look outside this directory
+    # at your own peril.
+    #
+    # @return [String] the full absolute path to the package root directory.
+    #
+    def package_dir
```

Comment:
I remember you mentioning fetch_dir.  I'm ok with that if you are.  I can s/package_dir/fetch_dir/g.